exercises/sum.cpp

```diff
@@ -1,8 +1,21 @@
-/*
-  Write a program that takes as input two numbers and print the sum.
+#include <iostream>
+using namespace std;
 
-  Output:
-  Insert the first number: 1
-  Insert the second number: 2
-  Sum: 3
-*/
+int sum(int a, int b){
+    return a+b;
+}
+
+int main(){
+    int x;
+    int y;
+    cout<<"Inserisci il primo numero:";
+    cin>>x;
+    cout<<endl;
+    cout<<"Inserisci secondo numero:";
+    cin>>y;
+    cout<<endl;
+    int out;
+    out = sum(x,y);
+    cout<<"Risultato: "<<out;
+    return 0;
```

Comment:
potresti sostituire 0 con EXIT_SUCCESS in modo da migliorare la leggibilità del codice
```c++
#define EXIT_SUCCESS 0
...
return EXIT_SUCCESS;
```